### ISSUE TITLE
OctoPi fails when trying to connect to Tevo Tarantula after first recv >

### ISSUE THREAD

**Issue by SyntaxWarrior** [foosel/OctoPrint#2365]
First of, thank you for provding this software. I am really looking forward to being able to use it.

**#### What were you doing?**
Trying to connect my Raspberry pi 3 (with updated version of OctoPrint as of today) with a Tevo Tarantula printer.

**Ideally provide exact steps to follow in order to reproduce your problem:**

1. Have USB Cable connected to a port of the Raspberry pi.
2. Power on Pi and Printer at same time (same switch in multisocket connector).
3. Log into OctoPrint.
4. Press Connect.
5. View messages in terminal
6. Terminal prints out:
"There was a timeout while trying to connect to the printer
Changing monitoring state from 'Connecting' to 'Offline'
Connection closed, closing down monitor"
After printer responds with:
```
Connecting to: /dev/ttyUSB0
Changing monitoring state from 'Offline' to 'Opening serial port'
Connected to: Serial<id=0x6e29c570, open=True>(port='/dev/ttyUSB0', baudrate=9600, bytesize=8, parity='N', stopbits=1, timeout=20.0, xonxoff=False, rtscts=False, dsrdtr=False), starting monitor
Changing monitoring state from 'Opening serial port' to 'Connecting'
Send: N0 M110 N0*125
Recv: �\x13\x14\x13�\x13�\x1by/\x13kr�#\x10P7�\x0f\x16��\x07\x17\x1b�\x17\x1f\x06\x1b\x1f'\x1b\x12�\x16�j�_1\x17\x12\x13����
There was a timeout while trying to connect to the printer
Changing monitoring state from 'Connecting' to 'Offline'
Connection closed, closing down monitor
```
7. Have tried all different available baud rates with same result (just more data being written after _**Recv:**_
8. Have tried Auto on both options.
9. Have tried increasing timeout values by 20 seconds, no difference (just takes longer).
10. As a side note I have been able to print 2 test cubes when connected to my Windows 10 Laptop with Repetier. They turned out well.

**#### What did you expect to happen?**
Some sort of Ok message and connection success, and interface saying State: Connected.

The interesting part here is that before I pressed update OctoPi actually connected so that I could connect the printer through the Control Tab. But now after update it never gets that far.

**#### What happened instead?**
The web interface returns to State: Offline. With Terminal Tab writing out: "There was a timeout while trying to connect to the printer"

**#### Did the same happen when running OctoPrint in safe mode?**
Yes.

**#### Version of OctoPrint**
OctoPrint 1.3.6

#### Operating System running OctoPrint
OctoPi 0.14.0

#### Printer model & used firmware incl. version
Tevo Tarantula. Says Migbot 3D printer on display. I have not touched printer firmware.
Have not figured out how to get the actual version of it yet.

#### Browser and Version of Browser, Operating System running Browser
Chrome Version 63.0.3239.132

#### Link to octoprint.log
https://pastebin.com/72ySKVC8

#### Link to contents of terminal tab or serial.log
baudrate=250000: https://pastebin.com/5dvVQXTQ
baudrate=9600: https://pastebin.com/Qm2md2ns

#### Link to contents of Javascript console in the browser
https://pastebin.com/6scFY9Kz

#### Screenshot(s)/video(s) showing the problem:
Video of me clicking through all the tabs (so you can see all the settings) and producing the above error.
https://www.youtube.com/watch?v=lpEndz7D4WM

I have read the FAQ.

**Comment by SyntaxWarrior** [comment]
Here are the log files from the Raspberry

[serial (3).log](https://github.com/foosel/OctoPrint/files/1629692/serial.3.log)
[serial (2).log](https://github.com/foosel/OctoPrint/files/1629693/serial.2.log)
[octoprint (1).log](https://github.com/foosel/OctoPrint/files/1629694/octoprint.1.log)

**Comment by SyntaxWarrior** [comment]
Got it working today. Well, got past this issue at least and am now stuck on a different problem. Printer times out. I'll bang my head at that for a while instead and see if I can get this sorted.. 

Not actually sure whats different but it seems to connect at 115200 baud and setting the usb port manually and not using auto. Very certain I tried this yesterday to but maybe I skipped this baud setting on accident.

Heres my current log if anyone finds that usefull.
`Connecting to: /dev/ttyUSB0
Changing monitoring state from 'Offline' to 'Opening serial port'
Connected to: Serial<id=0x710d4ab0, open=True>(port='/dev/ttyUSB0', baudrate=115200, bytesize=8, parity='N', stopbits=1, timeout=20.0, xonxoff=False, rtscts=False, dsrdtr=False), starting monitor
Changing monitoring state from 'Opening serial port' to 'Connecting'
Send: N0 M110 N0*125
Recv: start
Send: N0 M110 N0*125
Recv: echo:Marlin1.0.0
Recv: echo: Last Updated: Oct 19 2017 15:23:20 | Author: Zviper89, Migbot Stndrdauto lvl
Recv: Compiled: Oct 19 2017
Recv: echo: Free Memory: 3596  PlannerBufferBytes: 1232
Recv: echo:Hardcoded Default Settings Loaded
Recv: echo:Steps per unit:
Recv: echo:  M92 X80.00 Y80.00 Z1600.00 E100.00
Recv: echo:Maximum feedrates (mm/s):
Recv: echo:  M203 X225.00 Y225.00 Z3.00 E25.00
Recv: echo:Maximum Acceleration (mm/s2):
Recv: echo:  M201 X2650 Y2650 Z100 E10000
Recv: echo:Acceleration: S=acceleration, T=retract acceleration
Recv: echo:  M204 S2650.00 T3000.00
Recv: echo:Advanced variables: S=Min feedrate (mm/s), T=Min travel feedrate (mm/s), B=minimum segment time (ms), X=maximum XY jerk (mm/s),  Z=maximum Z jerk (mm/s),  E=maximum E jerk (mm/s)
Recv: echo:  M205 S0.00 T0.00 B20000 X15.00 Z0.40 E5.00
Recv: echo:Home offset (mm):
Recv: echo:  M206 X0.00 Y0.00 Z0.00
Recv: echo:PID settings:
Recv: echo:   M301 P22.20 I1.08 D114.00
Recv: echo:SD card ok
Recv: ok
Changing monitoring state from 'Connecting' to 'Operational'
Send: N0 M110 N0*125
Recv: ok
Send: N1 M115*39
Recv: FIRMWARE_NAME:Marlin V1; Sprinter/grbl mashup for gen6 FIRMWARE_URL:https://github.com/ErikZalm/Marlin/ PROTOCOL_VERSION:1.0 MACHINE_TYPE:Migbot 3D EXTRUDER_COUNT:1 UUID:00000000-0000-0000-0000-000000000000
Recv: ok
Send: M20
Recv: Begin file list
Recv: CUBE1.GCO
Recv: CUBE2.GCO
Recv: CUBE3.GCO
Recv: CUBE4.GCO
Recv: CUBE5.GCO
Recv: CUBE6.GCO
Recv: End file list
Recv: ok
Send: M105
Recv: ok T:17.2 /0.0 B:16.9 /0.0 T0:17.2 /0.0 @:0 B@:0
Send: M105
Recv: ok T:17.1 /0.0 B:16.8 /0.0 T0:17.1 /0.0 @:0 B@:0
Send: G91
Recv: ok
Send: G1 X10 F1000
Recv: ok
Send: G90
Recv: ok
Send: G91
Recv: ok
Send: G1 X10 F1000
Recv: ok
Send: G90
Recv: ok
Send: M105
Recv: ok T:16.9 /0.0 B:16.5 /0.0 T0:16.9 /0.0 @:0 B@:0
Send: G91
Recv: ok
Send: G1 X10 F1000
Recv: ok
Send: G90
Recv: ok
Send: G91
Recv: ok
Send: G1 X10 F1000
Recv: ok
Send: G90
Recv: ok
Send: G91
Recv: ok
Send: G1 X10 F1000
Recv: ok
Send: G90
Recv: ok
Send: G91
Recv: ok
Send: G1 X10 F1000
Recv: ok
Send: G90
Recv: ok
Send: M105
Recv: ok T:17.1 /0.0 B:16.7 /0.0 T0:17.1 /0.0 @:0 B@:0
Send: G91
Recv: ok
Send: G1 Z10 F400
Recv: ok
Send: G90
Recv: ok
Send: G91
Recv: ok
Send: G1 Z10 F400
Recv: ok
Send: G90
Recv: ok
Send: M105
Recv: ok T:17.0 /0.0 B:16.8 /0.0 T0:17.0 /0.0 @:0 B@:0
Send: G91
Recv: ok
Send: G28 X0 Y0
Recv: ok
Send: G90
Recv: ok
Send: M105
Recv: ok T:17.8 /0.0 B:17.4 /0.0 T0:17.8 /0.0 @:0 B@:0
Send: M105`
